### ISSUE TITLE
make WikiPathways class

### ISSUE THREAD

**Issue by MartijnAB** [MartijnAB/from-toxicological-data-to-toxicological-information#14]

**Comment by MartijnAB** [comment]
zo als vaak met python en ids gaan in imports fout ziet u zo snel de goed manier?

https://github.com/MartijnAB/from-toxicological-data-to-toxicological-information/blob/master/main/main.py

**Comment by MartijnAB** [comment]
het lijkt mij een goed idee om ook de labels (zoals "Codeine-6-glucuronide") te bewaren zodat ze ook in de nootbook gebruikt kunnen worden. Wat vind u?

**Comment by mkutmon** [comment]
> zo als vaak met python en ids gaan in imports fout ziet u zo snel de goed manier?

hmm, no

**Comment by MartijnAB** [comment]
Voor nu is het opgelost, want voor nu werk ik van uit de main functie in het bestand zelf.

**Comment by MartijnAB** [comment]
> > zo als vaak met python en ids gaan in imports fout ziet u zo snel de goed manier?
> 
> hmm, no

@mkutmon 

Voor nu is het opgelost, want voor nu werk ik van uit de main functie in het bestand zelf.

**Comment by MartijnAB** [comment]
> yes, but that will be part of the chemical compound class, right?

@mkutmon 

Ja maar het aan maken van de the chemical compound class gebuurt in de WikiPathways class; van daat de vraag.

**Comment by MartijnAB** [comment]
@mkutmon 

puur op functionaliteit is de class af.

om het regel 60 in het bestand:
https://github.com/MartijnAB/from-toxicological-data-to-toxicological-information/blob/master/main/model/WikiPathways.py#L67
zonder exit() te laten verlopen zal er een error in moeten, ik denk dat dat netter is, ook naar de gebruiker; wat denkt u?

**Comment by mkutmon** [comment]
yeah, I think it's important to tell the user what happened and that no Pathway will be returned!
